openapi/paths/customers@{id}@edd-timeline@{messageId}.yaml

```
@@ -0,0 +1,50 @@
+parameters:
+  - $ref: ../components/parameters/resourceId.yaml
+  - name: messageId
+    in: path
+    description: EDD timeline message ID.
```

nitpick: based on the style
```suggestion
    description: ID of the EDD timeline message.
```

---

I copy pasted from https://github.com/Rebilly/api-definitions/blob/4cdf5b21c53ee08f6d58c7abf993cdde0ded9046/openapi/paths/customers%40%7Bid%7D%40timeline%40%7BmessageId%7D.yaml#L5

---

It seems we still have some inconsistencies on this. I'll make a PR